dart/lib/src/open_location_code.dart

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
_This is an automated bot comment from the TravisCI tests_<br>Test formatting (br,pre) <br><pre>6c6  
< var bar = (47/5).toInt();  
---  
> var bar = (47 / 5).toInt();<pre>